### PR TITLE
Cut 2.18.0-rc.1

### PR DESCRIPTION
**Pull request by bwplotka**
One more item added to CHANGELOG, bugfix we fixed in https://github.com/prometheus/prometheus/pull/7182

image

Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+## 2.18.0-rc.1 / 2020-04-30
 
 * [CHANGE] Federation: Only use local TSDB for federation (ignore remote read). #7077
 * [CHANGE] Rules: `rule_evaluations_total` and `rule_evaluation_failures_total` have a `rule_group` label now. #7094
 * [FEATURE] Tracing: Added experimental Jaeger support #7148
 * [ENHANCEMENT] TSDB: Significantly reduce WAL size kept around after a block cut. #7098
 * [ENHANCEMENT] Discovery: Add `architecture` meta label for EC2. #7000
+* [BUGFIX] API/UI: Fixed wrong MinTime reported by /status. #7182
 * [BUGFIX] React UI: Fixed multiselect legend on OSX. #6880
+* [CHANGE] Federation: Only use local TSDB for federation (ignore remote read). #7077
```
Comment (roidelapluie) [inline CHANGELOG.md:10]: Why are you re-adding this?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
-## 2.18.0-rc.0 / 2020-04-24
+## 2.18.0-rc.1 / 2020-04-30
 
 * [CHANGE] Federation: Only use local TSDB for federation (ignore remote read). #7077
 * [CHANGE] Rules: `rule_evaluations_total` and `rule_evaluation_failures_total` have a `rule_group` label now. #7094
 * [FEATURE] Tracing: Added experimental Jaeger support #7148
 * [ENHANCEMENT] TSDB: Significantly reduce WAL size kept around after a block cut. #7098
 * [ENHANCEMENT] Discovery: Add `architecture` meta label for EC2. #7000
+* [BUGFIX] API/UI: Fixed wrong MinTime reported by /status. #7182
```
Comment (roidelapluie) [inline CHANGELOG.md:8]: Maybe specify that it is for tsdb?

File: CHANGELOG.md (hunk 1 above)
Comment (bwplotka) [reply CHANGELOG.md:10]: copy pasta

File: CHANGELOG.md (hunk 2 above)
Comment (bwplotka) [reply CHANGELOG.md]: The bug was only on API level though (:

Comment (roidelapluie) [reply CHANGELOG.md]: Only on UI level

**Comment by brian-brazil** [comment]
:+1:

**Comment by cstyan** [comment]
:+1:

**Pull request merged**